### DISCUSSION TITLE
yarn dev not found

### DISCUSSION THREAD

**Discussion by blissdev** [pmndrs/jotai#1946] [General] [answered]
Following the contributing guidelines which told me to run `yarn dev` to render the docs, I received the following error message.

```
│ yarn dev
yarn run v1.22.19
error Command "dev" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

Should I be doing something else or is that broken?

**Comment by arjunvegda** [comment, marked as answer]
Are you running this command within the `website` directory? (Please see step [#1 in CONTRIBUTING.md#docs](https://github.com/pmndrs/jotai/blob/main/CONTRIBUTING.md#docs))

**Reply by blissdev** [reply]
Oh I’m silly I was running it from docs. Thanks!
